src/uwtools/tests/utils/test_file.py

```diff
@@ -103,7 +133,7 @@ def test_readable_file(tmp_path):
 
 def test_readable_nofile():
     with file.readable() as f:
-        assert f is sys.stdin
+        assert hasattr(f, "read")
```

Comment:
The object yielded from `readable()` will no longer **be** `stdin`, but will be `read()`-able.